webclient/app/src/app/features/apps/appSteps/AppConclusion.jsx

```diff
@@ -106,7 +106,7 @@ function AppConclusion(props) {
             <Typography variant={"h4"} gutterBottom>
                 {t("app.section.entityDiagram")}
             </Typography>
-            <EntityDiagram entities={app.entities} appId={app.id} coordinates={coordinates} dense editable={false}/>
+            <EntityDiagram entities={app.entities} dense editable={false}/>
```

Comment:
Ich nehme einfach an, dass appId nicht zwingend notwendig ist, wenn editable auf false ist? Eigentlich sollte das nichts sein, ich frag nur nochmal nach :)

Reply:
ich habe die Checks auf editable for den DB-Aufrufen (und damit auch der Verwendung der appId) nochmal eingebaut.